src/main/java/mainclass/exercises/ArrayManipulations.java

```diff
@@ -0,0 +1,31 @@
+package mainclass.exercises;
+
+import java.util.Random;
+
+public class ArrayManipulations {
+    public static void main(String[] args) {
+        Random random = new Random();
+        int[] array = random.ints(5, 1, 5).toArray();
+        for (int i = 0; i < array.length; i++) {
+            System.out.println(array[i]);
+        }
+        int sum = array[0];
+        int largest = array[0];
```

Comment:
Variable sum is equal to variable largest, or use one variable, or "sum" should receive another value.